esp32/modules/easyrtc.py

```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
```

Comment:
Let's try and make PR's do seperate things :p 
If you've already committed something on a branch you're going to make a pr out of, you can still revert and pull it to another branch